server/channels/app/export.go

```diff
@@ -878,14 +884,38 @@ func (a *App) exportAllDirectChannels(ctx request.CTX, job *model.Job, writer io
 		for _, channel := range channels {
 			afterId = channel.Id
 
+			// Skip deleted.
+			if channel.DeleteAt != 0 {
+				continue
+			}
```

Comment:
Not sure about this. Exporting archived channels makes sense to me. They are even visible, depending on `ExperimentalViewArchivedChannels` setting.

Reply:
It was there before, funnily enough you can't actually delete GMs or DMs 🤷 So I though it may have some legacy reasons to keep it there.

Reply:
I believe this is wrong. Because `includeArchivedChannels` is a param passed to the function which is respected in the SQL query. So the channels contain archived channels depending on that flag, therefore there shouldn't be a reason to check for `DeleteAt` again. But it's unrelated to this PR, I agree.